.gitignore

```diff
@@ -48,6 +48,11 @@ terraform.rc
 *.pem
 *.p12
 *.pub
+*.pfx
+*.crt
 
 #Licenses
 *.license
+
+#Private key
+privatekey.key
```

Comment:
what if the key is named something different like privateKey or private_key

Reply:
You are missing an empty line at the end of the file.

Reply:
Added